chia/rpc/wallet_rpc_api.py

```diff
@@ -4496,3 +4504,83 @@ class CRCATApprovePending(Streamable):
         return {
             "transactions": [tx.to_json_dict_convenience(self.service.config) for tx in txs],
         }
+
+    ##########################################################################################
+    # VAULT
+    ##########################################################################################
+    @tx_endpoint(push=True)
```

Comment:
Let's start making these False by default, the ones with push=True currently are just for backwards compatbility